### PR TITLE
Api

### PR DESCRIPTION
**Pull request by jhallard**
Moved DevOps out of main Praxyk directory, cleaned up empty pod_container top-level directory. Added build scripts to queue route, added instant-deploy functionality to API server.

### CODE REVIEW

File: api/templates/credit_card_charge_failed_email.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<h4>Praxyk: Credit Card Charge Failure</h4> 
+<p>Hi {{ user_name }},</p>
+<p>Unfortnately your payment was unable to go through due to the credit card being declined. Your account will be suspended until you fix you information on file. You will still be able to login, but you will not be able to use our servies until you fix this problem.</p>
```
Comment (nsokolnikov) [inline api/templates/credit_card_charge_failed_email.html]: typo

Comment (jhallard) [inline api/templates/credit_card_charge_failed_email.html]: @rcoley93 <-- blame

Comment (jhallard) [inline api/templates/credit_card_charge_failed_email.html]: @nsokolnikov fixed here : https://github.com/Praxyk/Praxyk/commit/62841553fe77cb9c2bba1f7695c6df91cfca26ba

File: api/libs/payment_route.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        user = User.query.get(id)
+        try:
+         if user.payment_info.card_id == None:
+            return jsonify({'code':400,'message':"There is not a card on file!"})
+            
+         customer = stripe.Customer.retrieve(user.payment_info.customer_id)
+         card = customer.sources.retrieve(user.payment_info.card_id).delete()
+         
+         if not card.deleted :
+            return jsonify({'code':400,'message':"There was a problem removing you card!"})
```
Comment (nsokolnikov) [inline api/libs/payment_route.py]: your

Comment (jhallard) [inline api/libs/payment_route.py]: bleh  @rcoley93

**Comment by jhallard** [comment]
Merging now before project is due.

**Pull request merged**